modules/network-observability-operator-install.adoc

```diff
@@ -34,17 +34,13 @@ This documentation assumes that your `LokiStack` instance name is `loki`. Using
 . Navigate to the *Flow Collector* tab, and click *Create FlowCollector*. Make the following selections in the form view:
 .. *spec.agent.ebpf.Sampling*: Specify a sampling size for flows. Lower sampling sizes will have higher impact on resource utilization. For more information, see the "FlowCollector API reference", `spec.agent.ebpf`.
 .. If you are using Loki, set the following specifications:
-... *spec.loki.enable*: Select the check box to enable storing flows in Loki.
-... *spec.loki.url*: Since authentication is specified separately, this URL needs to be updated to `https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network`. The first part of the URL, "loki", must match the name of your `LokiStack`.
-... *spec.loki.authToken*: Select the `FORWARD` value.
-... *spec.loki.statusUrl*: Set this to `https://loki-query-frontend-http.netobserv.svc:3100/`. The first part of the URL, "loki", must match the name of your `LokiStack`.
+... *spec.loki.mode*: Set this to `LokiStack` mode.
+... *spec.loki.lokistack.name*: Set this to `LokiStack`.
```

Comment:
Here we are pointing the `LokiStack` instance [previously created from yaml](https://docs.openshift.com/container-platform/4.14/network_observability/installing-operators.html#network-observability-lokistack-create_network_observability):
```
  apiVersion: loki.grafana.com/v1
  kind: LokiStack
  metadata:
    name: loki
    namespace: netobserv 
...
```

The name should be `loki` as:
```suggestion
... *spec.loki.lokistack.name*: Set this to `loki`.
```

Since the namespace is `netobserv` (the same as our operator), we don't need to mention `spec.loki.lokistack.namespace`. However, we should add a note that this can be updated as you did in LokiStack creation:
```
The installation examples in this documentation use the same namespace, netobserv, across all components. You can optionally use a different namespace.
```

Reply:
@skrthomas if it is helpful, here are some examples of the `v1beta2` CRD
- official example in upstream operator repo: https://github.com/netobserv/network-observability-operator/blob/main/config/samples/flows_v1beta2_flowcollector.yaml
- QE CRD used in Perf pipeline: https://github.com/openshift-qe/ocp-qe-perfscale-ci/blob/netobserv-perf-tests/scripts/netobserv/flows_v1beta2_flowcollector.yaml

Reply:
@nathan-weinberg I was wondering where I got `LokiStack` from for thhe `spec.loki.lokistack.name` value, and I think I was looking at the 2nd CRD you linked; I think I found it in Jira. Anyways, the `spec.loki.lokistack.name` is `lokistack`...so is this wrong? Or do you guys just use `lokistack`
![image](https://github.com/openshift/openshift-docs/assets/77501157/8c25a056-82f8-4e73-bde4-cfd5194b3961)

Reply:
we just used lokistack :sweat_smile: but it could be anything really

Reply:
Got it! 😆

Reply:
QE guys doesn't follow the docs :smiling_imp:

Reply:
yeah we have been using name `lokistack` since the start but the docs and upstream mention `loki`. we dont follow the docs🫣

Reply:
I won't hold your feet to the fire 🤣  I think you can use what you want. Just for my personal information, I should remember this little naming thing when I'm looking at QE CRDs for doc updates